lib/state-controller.js

```diff
@@ -76,66 +69,60 @@ const StateController = {
   downloadKite(url, opts) {
     opts = opts || {};
     return this.canInstallKite()
-    .then(() => utils.spawnPromise('curl', [
-      '-L', url, '--output', this.KITE_DMG_PATH,
-    ], 'curl_error'))
+    .then(() => this.streamKiteDownload(opts.onDownloadProgress))
     .then(() => utils.guardCall(opts.onDownload))
     .then(() => opts.install && this.installKite(opts));
   },
 
-  installKite(opts) {
-    opts = opts || {};
+  streamKiteDownload(progress) {
```

Comment:
yup makes sense. we should still make sure that it downloads correctly on a VM, which was the only case that broke this i believe